PinClustering/PinClustering/ViewControllers/MapViewController+MapDataSourceExtension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MapViewController+MapDataSourceExtension.swift
+//  PinClustering
+//
+//  Created by Norbert Agoston on 13/09/16.
+//  Copyright © 2016 Norbert Agoston. All rights reserved.
+//
+
+import Foundation
+
+extension MapViewController {
+    func updatePointsOfInterestWith(locations: NSArray) {
+        for locationInfo in locations {
```

Comment:
try to use something more specific to swift. You could use either map, enumarate or foreach like this.

for(index,location) in location.enumarate() {}
locations.forEach { locationInfo in }
or the best one
self.pointsOfInterest = locations.map { locationInfo in return FuelLocation(locationInfo: locationInfo as? [String : AnyObject]) }